test/Test/Bridge/Repository/UserRepositoryTest.php

```diff
@@ -100,4 +101,20 @@ public function testFindOneByEavAttribute(): void
         $user = $this->repository->findOneByShippingState('North Dakota');
         self::assertSame(4, $user->id);
     }
+
+    public function testFieldMapping(): void
+    {
+        $repository = new UserRepository();
+        $repository->setEntityManager(new class extends EntityManager {
+            public function __construct() {}
+            public function getTablesPrefix(): string { return 'foo_'; }
+        });
+
+        $this->assertEquals('billing_address_1', $repository->getMappedMetaKey('billing_address_1'));
+        $this->assertEquals('billing_address_2', $repository->getMappedMetaKey('billing_address_2'));
+        $this->assertEquals('shipping_address_1', $repository->getMappedMetaKey('shipping_address_1'));
+        $this->assertEquals('shipping_address_2', $repository->getMappedMetaKey('shipping_address_2'));
+        $this->assertEquals('foo_capabilities', $repository->getMappedMetaKey('capabilities'));
```

Comment:
Wasn't really sure how to test this. Ideally it's probably best to duplicate `testFindReturnsCorrectUser` but prepare a scenario where table prefixes are used in WP. That seems a bit overkill though for this particular case. Although it would probably useful for the project as a whole to have such tests.

Reply:
This test is good enough, thanks!